src/mbgl/style/sources/vector_source.cpp

```diff
@@ -61,7 +63,8 @@ void VectorSource::loadDescription(FileSource& fileSource) {
                 observer->onSourceError(*this, std::make_exception_ptr(util::StyleParseException(error.message)));
                 return;
             }
-
+            if (maxZoom) tileset->zoomRange.max = *maxZoom;
```

Comment:
Do we need same fix in glsj? I can see that in gjs minzoom and maxzoom are set to 0, 22 respectively.

Reply:
No, js works fine, the `maxzoom` option is passed to `tileJson` while function`loadTileJson`. That's why the customer realized the differences.